mlir-tensorrt/tensorrt/test/Dialect/TensorRT/reshape-elimination.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: tensorrt-opt %s -split-input-file  -tensorrt-reshape-elimination | FileCheck %s
+// RUN: tensorrt-opt %s -split-input-file  -tensorrt-transpose-reshape-elimination | FileCheck %s
```

Comment:
@christopherbate I have updated the lit test for reshape and transpose to use the new `transpose-reshape-elimination` pass